### PR TITLE
Enable CodeQL for scheduled CI jobs

### PR DESCRIPTION
**Pull request by kurtzeborn**

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]
Once test pipelines are done this looks good.

File: eng/pipelines/templates/stages/archetype-sdk-client.yml [hunk 1]
```diff
@@ -73,6 +73,9 @@ parameters:
   type: boolean
   default: false
 
+variables:
+  - template: ../variables/globals.yml
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/archetype-sdk-client.yml:77]:
```suggestion
  - template: /eng/pipelines/templates/variables/globals.yml
```
nit: Not necessary but makes copying around easier if this is relative to the root.

File: eng/pipelines/templates/stages/cosmos-sdk-client.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@ parameters:
   type: boolean
   default: false
 
+variables:
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/cosmos-sdk-client.yml:21]: For this case it looks like they are already including our shared client so it shouldn't be needed here.

Comment (weshaggard) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml:21]: In fact you cannot add it see https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2188656&view=results

Comment (kurtzeborn) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: I've taken it out.

Comment (weshaggard) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: @kurtzeborn in order to fix the current cosmos issue you might need to update to use extends in the cosmos-sdk-client.yml. Only question is if you can do a double extends.

Comment (weshaggard) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: You would need to attempt to change https://github.com/Azure/azure-sdk-for-python/blob/main/eng/pipelines/templates/stages/cosmos-sdk-client.yml#L21 to extends. However I'm not sure if that will work and even if it does work I'm not sure how that will work when they re-enable their emulator tests which are currently commented out.

Comment (kurtzeborn) [reply eng/pipelines/templates/stages/cosmos-sdk-client.yml]: I think we fixed it now.  Thanks for the help!

**Pull request merged**